specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```
@@ -0,0 +1,1192 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Anomaly Detector Client",
+    "version": "v1.1",
+    "description": "Multivariate Time-Series Anomaly Detection API"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Ocp-Apim-Subscription-Key",
+      "description": "The secret key for your Azure Anomaly Detector subscription."
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/multivariate/detect-batch/{resultId}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Get multivariate anomaly detection result by Result Id.",
+            "schema": {
+              "$ref": "#/definitions/DetectionResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Get Multivariate Anomaly Detection Result",
+        "description": "For asynchronous inference, get multivariate anomaly detection result based on resultId returned by the /models/<string:modelId>/entire/detect api.",
```

Need to update this comment. I recommend using the operationId rather than the path so that comments in the generated SDKs will be better.
```suggestion
        "description": "For asynchronous inference, get multivariate anomaly detection result based on resultId returned by the BatchDetectAnomaly api.",
```

---

fixed